mobile/src/modal.js

```diff
@@ -33,7 +33,15 @@ export default class CustomModal extends Component {
       inputHeight: 0,
       search: false,
       session: '',
-      newList: []
+      newList: [],
+      isError: this.props.showError
```

Comment:
Why turn this prop into state?  Can it be left as a prop?

Reply:
This was for a fix related to dropping the error help text as soon as a user types. To do that we need to store the showError prop locally as the input is only stored locally until submitted